src/main/java/io/lumigo/handlers/LumigoRequestHandler.java

```diff
@@ -73,4 +75,16 @@ public OUTPUT handleRequest(INPUT input, Context context) {
     }
 
     public abstract OUTPUT doHandleRequest(INPUT input, Context context);
+
+    public void addExecutionTag(String key, String value) {
```

Comment:
I talked with Harel and we agreed that we dont want to add this here, lets expose those methods from the `ExecutionTags` class.